src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -187,24 +203,23 @@ private Meal pickRandomMeal(List<Meal> mealList) {
         return (Meal)mealList.get(index);
     }
 
-    private Boolean isResultOfSearch(Meal meal, List<String> params) {
+    private int getFrequency(Meal meal, List<String> params) {
+        int frequency = 0;
         for (String param : params) {
-            if (meal.getTitle().contains(param)) {
-                return true;
-            } 
-            if (meal.getDescription().contains(param)) {
-                return true;
+            if (meal.getTitle().contains(param) ||
+                meal.getDescription().contains(param) ||
+                meal.getType().contains(param)) {
+                frequency++;
+                break;
```

Comment:
This may be just my opinion, but I tend to avoid `break` and replace them with more explicit data flow operations whenever possible (like `if ... else`, for example).
In Java, in most cases you can refactor your code not to use loops (and, in consequence, `break`) by using the [Stream](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html) feature. I understand why it may feel like an overcomplicated approach (a `for` loop seems simpler, at a first glance), but in some cases it really pays off. (long story short, it is, in my opinion, far easier to write a for loop with a bug than a `Stream` chain with a bug)

Feel free to ignore the Streams suggestion, though. I just wanted to point out that there's usually many approaches to solve the same problem, and it's often useful to consider all of them.

However, I believe this block of code may have a tiny bug that may have been avoided had we used streams.